pkgs/tools/networking/mcrcon/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub
+, gcc }:
+
+stdenv.mkDerivation rec {
+  name = "mcrcon-${version}";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "Tiiffi";
+    repo = "mcrcon";
+    rev = "v${version}";
+    sha256 = "1pwr1cjldjy8bxqpp7w03nvdpw8l4vqfnk6w6b3mf0qpap1k700z";
+  };
+
+  buildInputs = [ gcc ];
+
+  buildPhase = ''
+    gcc mcrcon.c -o mcrcon
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m 755 mcrcon $out/bin
+  '';
+
+  phases = "unpackPhase buildPhase installPhase fixupPhase";
```

Comment:
Thanks for the review! I hoped excluding some could speed up the build. Not a good practice?